tests/unit/components/sl-chart-test.js

```diff
@@ -0,0 +1,363 @@
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testSeries was empty'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: "string"
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testSeries was string'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: undefined
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testSeries was undefined'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: true
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testSeries was a boolean'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: 666
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testSeries was number'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: function(){}
+        });
+    }  catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        assertionThrown,
+        'testSeries was a function'
+    );
+
+    assertionThrown = false;
+
+    try {
+        this.subject({
+            options: testOptions,
+            series: testSeries
+        });
+    } catch( error ) {
+        assertionThrown = true;
+    }
+
+    assert.ok(
+        !assertionThrown,
+        'testSeries was an Array'
+    );
+});
+
+test( "Style method returns correct string", function( assert ) {
+    let component = this.subject({
+        options: testOptions, 
+        series: testSeries
+    });
+
+    assert.equal( component.get( 'style' ).string, 'height: auto; width: auto;' );
+
+    component.set( 'height', 10 );
+
+    assert.equal( component.get( 'style' ).string, 'height: 10; width: auto;' );
+});
+
+test( "Title property is set", function( assert ) {
+    let component = this.subject({
+        options: testOptions, 
+        series: testSeries
+    });
+
+    assert.equal( component.get( 'title' ), null );
+
+    component.set( 'title', 'Peak server hours' );
+
+    assert.equal( component.get( 'title' ), 'Peak server hours' );
+});
+
+test( "Chart property is set after setupChart() is ran", function( assert ) {
+    let component = this.subject({
+        options: testOptions, 
+        series: testSeries
+    });
+
+    assert.equal( component.get( 'chart' ), null);
+
+    this.render();
+
+    assert.ok( component.get( 'chart' ), 'chart is rendered' );
```

Comment:
If were keeping this tests, would need to have modified whitespace to conform to https://github.com/softlayer/ember-style-guide#whitespace